cdxev/__main__.py

```diff
@@ -712,6 +714,50 @@ def create_build_public_bom_parser(
     return parser
 
 
+def create_list_command_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "list",
+        help=(
+            "Lists specific contents of the SBOM."
+            "Currently supported are the listing of license information and component information"
+        ),
+        usage=(
+            "cdx-ev list [-h] [--format {txt,csv}] [--skip-metadata] "
+            "[--output <file>] <operation> input"
+        ),
+    )
+    parser.add_argument(
+        "operation",
+        metavar="<operation>",
+        help=("The list operation that shell be performed"),
```

Comment:
"shall" and dot at the end of the sentence

Reply:
done